src/components/forms/TaggerRemoveComponent.jsx

```diff
@@ -0,0 +1,13 @@
+import React, {PropTypes} from 'react';
+
+import Icon from '../Icon.jsx';
+
+export default class TaggerRemoveComponent extends React.Component {
+  render() {
+    return (
+      <button {...this.props}>
+        <Icon color="#a00" name="fa-times-circle" />
```

Comment:
I'm going to try and extract all the colors out of literals into the `/src/settings.js` file. This way we don't get different versions of red and such. I might change `#a00` to `settings.warningColor` unless there was a reason for this color. In that case, ignore this comment.